pkg/resources/custom_diffs.go

```diff
@@ -2,51 +2,74 @@ package resources
 
 import (
 	"context"
+	"log"
 	"strconv"
 
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/customdiff"
+
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/helpers"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/provider"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/customdiff"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-// NestedIntValueAccountObjectComputedIf is NestedValueComputedIf,
-// but dedicated for account level objects with integer-typed properties.
-func NestedIntValueAccountObjectComputedIf(key string, parameter sdk.AccountParameter) schema.CustomizeDiffFunc {
-	return NestedValueComputedIf(
-		key,
-		func(client *sdk.Client) (*sdk.Parameter, error) {
-			return client.Parameters.ShowAccountParameter(context.Background(), parameter)
-		},
-		func(v any) string { return strconv.Itoa(v.(int)) },
-	)
+func AccountObjectStringValueComputedIf(key string, params []*sdk.Parameter, parameterLevel sdk.ParameterType, parameter sdk.AccountParameter) schema.CustomizeDiffFunc {
```

Comment:
Yeah, it was mostly referring to the fact that it's an account object, because I wasn't sure if all of the parameters will be "inheirated" and defined in `sdk.AccountParameter` it could be renamed. But I think `sdk.AccountParameter` will always have needed values and can be used for objects other than account objects, so Removed the AccountObject prefix